src/Pages/Main/Components/StoreDetail/Modal.js

```diff
@@ -51,6 +26,43 @@ const Modal = props => {
     ? 1.0
     : 0;
 
+  useEffect(() => {
+    let post = { ...postData };
+    post['rating'] = rating;
+    setPost(post);
+  }, [starFilled]);
+
+  const handleImgLink = e => {
+    let post = { ...postData };
+    post['image_url'] = e.target.value;
+    setPost(post);
+  };
```

Comment:
추가적으로 스프레드 연산자를 사용한다면
변수를 선언하지 않고 아래와 같이 가능할 것 같습니다.
다른 함수도 이와 같이 수정할 수 있을 것 같네요!
```suggestion
  const handleImgLink = e => {
    setPost({ ...postData, image_url: e.target.value })
  };
```

Reply:
Aㅏ... 이런 방법이...!